database/dbase.go

```diff
@@ -109,8 +109,8 @@ func ConfigureDbDefault(v cs.Configuration, prefix, envUser, envPasswd string) {
 	v.SetDefault(prefix+"-migration-version", "")
 	v.SetDefault(prefix+"-connection-check", true)
 
-	v.BindEnv(prefix+"-username", envUser)
-	v.BindEnv(prefix+"-password", envPasswd)
+	_ = v.BindEnv(prefix+"-username", envUser)
+	_ = v.BindEnv(prefix+"-password", envPasswd)
```

Comment:
?

Reply:
Reverted to process it correctly when we'll have time